### PR TITLE
Add deprecation notice for 6M flash devices

### PR DESCRIPTION
**Pull request by grische**
## Description

6M flash devices are not future-proof and have been dropped by some communities already.

## Motivation and Context

https://github.com/freifunk-darmstadt/gluon-firmware-selector/pull/159

## How Has This Been Tested?

N/A

## Checklist:

- [ ] My code follows the code style of this project. (CI will test it anyway and also needs approval)
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: lib/config_default.js [hunk 1]
```diff
@@ -200,9 +200,21 @@ export const config = {
     "AP121U",
     "D-Link DIR-615",
     "D-Link DIR-615 D",
+    "D-Link DIR-825",
```
Comment (blocktrron) [inline lib/config_default.js:203]: Why? This has been fixed and the limit only applies to factory installations (Which are removed from Gluon)

Comment (grische) [reply lib/config_default.js:203]: I went through the list of targets of the latest release and it was marked as only 6M flash 
https://github.com/freifunk-gluon/gluon/blob/7feebe96a6b93d567bd342ffd01abcb3022383c5/targets/ath79-generic#L124

Comment (blocktrron) [reply lib/config_default.js:203]: This is outdated then, see https://github.com/openwrt/openwrt/commit/aca8bb5cc332f0ffdf4249e76b0a56716f98bef0

**Comment by grische** [comment]
@blocktrron can we merge this PR?

**Comment by skorpy2009** [comment]
@grische please rebase

**Comment by grische** [comment]
Let's focus on this PR then.

**Pull request closed without merging**